sympy/combinatorics/homomorphisms.py

```diff
@@ -421,7 +421,7 @@ def block_homomorphism(group, blocks):
     H = GroupHomomorphism(group, codomain, images)
     return H
 
-def group_isomorphism(G, H, isomorphism=True):
+def group_isomorphism(G, H, isomorphism=True, epimorphism=False, all=False):
```

Comment:
It shouldn't be called "isomorphism" if we are allowing it to compute epimorphisms as well. That's why I suggested calling it "homomorphism" - though I guess we have that one. Perhaps `find_homomorphism` since we are not defining a specific one here, but asking for any one? And it would make sense to give that keywords `injective` and `surjective` because that would give you the most versatility. And what used to be `isomorphism` could actually become something like `check` (so that `check=True` means only check existence, don't actually compute the homomorphism)